src/MarkdownEditor.js

```diff
@@ -44,12 +41,28 @@ var MarkdownEditor = React.createClass({
   render: function() {
     var divContent;
     var editorMenu;
+    var styles;
+    if(!this.props.styles) {
```

Comment:
Would it be better to use [defaultProps?](https://facebook.github.io/react/docs/react-component.html#defaultprops) 

Just add some spaces around the colon and the empty object, and between the if and opening parenthesis